Source/WebCore/platform/graphics/nicosia/NicosiaPaintingEngine.cpp

```diff
@@ -36,8 +36,7 @@ namespace Nicosia {
 
 std::unique_ptr<PaintingEngine> PaintingEngine::create()
 {
-#if (ENABLE(DEVELOPER_MODE) && PLATFORM(WPE)) || USE(GTK4)
```

Comment:
This also removes `ENABLE(DEVELOPER_MODE)`, so it means that this commit does not only affects WPE.
It also enables threaded rendering by default for GTK on production builds (cmake), while it was only enabled for developer builds before (build-webkit).